quickwit/quickwit-query/src/elastic_query_dsl/range_query.rs

```diff
@@ -81,3 +94,15 @@ impl ConvertableToQueryAst for RangeQuery {
         Ok(ast.boost(boost))
     }
 }
+
+
+fn parse_and_convert(value: JsonLiteral, fmt: &str) -> anyhow::Result<JsonLiteral> {
```

Comment:
I think there is benefit to parsing the time input format in caller.
```suggestion
fn parse_and_convert(value: JsonLiteral, date_format: &DateTimeInputFormat) -> anyhow::Result<JsonLiteral> {
```

Reply:
Great point! it's a shame that I didn't notice it myself. Thanks!